python/stressng/README.md

```diff
@@ -0,0 +1,19 @@
+# Stressng workload plugin for Arcaflow
+
+arca-stressng is a workload plugin for the [stressng](https://github.com/ColinIanKing/stress-ng) benchmark tool. It is using the [Arcaflow python SDK](https://github.com/arcalot/arcaflow-plugin-sdk-python).
+
+The supported workflow parameters (not all are implemented yet) and the working stressors can be found in the `WorkloadParams` schema of the [stressng_plugin.py](stressng_plugin.py) file. 
+Define your paramaters in the YAML file to be passed to the plugin. 
+
+## To test
```

Comment:
add a containerized section

Reply:
I've taken the official README now, hopefully that's the intended way.